app/src/main/java/com/gdg/android/MainActivity.kt

```diff
@@ -123,10 +138,171 @@ fun ProfileScreen(name: String, major: String, modifier: Modifier = Modifier) {
     }
 }
 
+@Composable
+fun LoginPage(navController: NavController? = null) {
+    val context = LocalContext.current
+
+    val majorTextValue = remember { mutableStateOf("") }
+    val nameTextValue = remember { mutableStateOf("") }
+    var majorError by remember { mutableStateOf("") }
+    var nameError by remember { mutableStateOf("") }
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(Color.White)
+            .padding(30.dp),
+        horizontalAlignment = Alignment.Start,
+        verticalArrangement = Arrangement.Center
+    ) {
+        Text(
+            text = "안녕하세요, 여러분",
+            fontSize = 24.sp,
+            fontWeight = FontWeight.Bold,
+            modifier = Modifier.fillMaxWidth(),
+            textAlign = TextAlign.Start
+        )
+
+        Spacer(modifier = Modifier.height(20.dp))
+
+        // 학부 입력
+        Text(
+            text = "학부",
+            fontSize = 16.sp,
+            color = Color(0xFF1C1C1C),
+            modifier = Modifier
+                .padding(top = 10.dp)
+                .fillMaxWidth()
+        )
+
+        Row(modifier = Modifier.fillMaxWidth()) {
+            Column(
+                modifier = Modifier.weight(1f)
+            ) {
+                TextField(
+                    value = majorTextValue.value,
+                    onValueChange = { majorTextValue.value = it },
+                    label = { Text("학부를 입력해주세요", color = Color.Black) },
+                    colors = TextFieldDefaults.colors(
+                        focusedContainerColor = Color(0xFFC4C4C4),
+                        unfocusedContainerColor = Color(0xFFE0E0E0),
+                        focusedIndicatorColor = Color.Transparent,
+                        unfocusedIndicatorColor = Color.Transparent,
+                        focusedTextColor = Color.Black,
+                        unfocusedTextColor = Color.Black,
+                        focusedLabelColor = Color.Black,
+                        unfocusedLabelColor = Color.Black
+                    ),
+                    shape = RoundedCornerShape(12.dp),
+                    modifier = Modifier
+                        .padding(top = 10.dp)
+                        .fillMaxWidth()
+                )
+
+                // 학부 입력 검증 메시지 표시
+                if (majorError.isNotEmpty()) {
+                    Text(text = majorError, color = Color.Red, fontSize = 12.sp, modifier = Modifier.padding(top = 4.dp))
+                }
```

Comment:
에러가 발생하는 경우 if 문이 실행되어 텍스트 필드 아래에 텍스트 컴포저블이 추가되므로 위치가 변동될 수 밖에 없습니다. 
위치를 고정시키려면 텍스트 크기와 동일한 크기의 여백을 미리 줘야 합니다.
```suggestion
                if (nameError.isNotEmpty()) {
                    Text(text = nameError, color = Color.Red, fontSize = 12.sp, modifier = Modifier.padding(top = 4.dp))
                } else {
                    Spacer(modifier = Modifier.height(12.dp))
                }
```